SConstruct

```diff
@@ -347,16 +402,19 @@ Default(t)
     #"test/checkpoint/workers.c"],
     #LIBS=[wtlib])
 
-t = env.Program("t_huge",
-    "test/huge/huge.c",
-    LIBS=[wtlib] + wtlibs)
-Default(t)
+#t = env.Program("t_huge",
+#    "test/huge/huge.c",
+#    LIBS=[wtlib] + wtlibs)
+#env.Alias("test", env.SmokeTest(t))
+#Default(t)
 
 t = env.Program("t_fops",
     ["test/fops/file.c",
     "test/fops/fops.c",
     "test/fops/t.c"],
     LIBS=[wtlib, shim] + wtlibs)
+env.Append(CPPPATH=["test/utility"])
+env.Alias("test", env.SmokeTest(t))
```

Comment:
I would like you to use "check" instead of "test".

The rest of the scons file uses check, and it is consistent with what Automake does.

See https://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets & http://www.gnu.org/software/automake/manual/html_node/Standard-Targets.html#Standard-Targets